packages/connect-query/src/stable-hash.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021-2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import stableHash from "stable-hash";
+
+/**
+ * Convert stable-hash to a typed API. This is a necessary workaround because
+ * stable-hash does not export a type for its default export that is compatible
+ * with Node16 resolution.
```

Comment:
attw reports "no types" for Node16 from ESM. https://arethetypeswrong.github.io/?p=stable-hash%400.0.3

That's a nice workaround, but is the JS file really part of `dist` after a build?

Reply:
Yep, tsc includes that file during a compile.